### ISSUE TITLE
Get-Credential does not work

### ISSUE THREAD

**Issue by Jaykul** [Jaykul/Jupyter-PowerShell#13]
Neither, for that matter does anything else which needs to prompt for input.

To get this working I need to write a proper "Host" so it's not the highest priority.

The workaround (for me, at least) is to use the [BetterCredentials](https://www.powershellgallery.com/packages/BetterCredentials/) and `-Store` the credentials from a regular PowerShell session so I can use them here.

**Comment by knoxyca** [comment]
The workaround that I've used (albeit insecure) is:
```
$Password = 'P@ssword'
$tUsername = "UserName"
$credential = New-Object -TypeName System.Management.Automation.PSCredential -ArgumentList $Username,$(convertTo-SecureString -AsPlainText -Force -String $Password)
```

I'm split as to if BetterCredentials in this case is insecure. Anyone with access to the Notebook will have access to the credentials of the user running the Notebook...

**Comment by Jaykul** [comment]
Well, access to a notebook (e.g. [readme.ipynb](https://github.com/Jaykul/Jupyter-PowerShell/blob/master/ReadMe.ipynb)) doesn't mean you have access to my computer, account and thus credentials stored in my Windows Credential Vault -- there are hundreds of ways I could share that document with you...

**Comment by knoxyca** [comment]
Good point. I was thinking more in the context of my having access to your jupyter instance. If I have access to your jupyter instance, then I have access to your information, but if I just have the notebook, then I don't really have the access (unless you take my workaround and store it in the notebook...).

For my use case the credential stored in the notebook is fine as the credentials aren't considered secure.
